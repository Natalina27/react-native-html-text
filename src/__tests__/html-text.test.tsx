import React from 'react';
import renderer from 'react-test-renderer';

describe('HtmlText component', () => {
    const HtmlText = require('../html-text/html-text').default;

    test('have exported a default class', () => {
        expect(HtmlText).toBeDefined();
    });

    test('renders correctly', () => {
        const tree = renderer.create(<HtmlText />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});