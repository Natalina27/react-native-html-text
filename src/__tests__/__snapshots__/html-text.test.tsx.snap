// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HtmlText component renders correctly 1`] = `
<Text>
  undefined
</Text>
`;
